deliver/lib/deliver/options.rb

```diff
@@ -181,6 +181,11 @@ def self.available_options
                                      description: "Clear all previously uploaded screenshots before uploading the new ones",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_unsupported_languages,
```

Comment:
Yeah, I also don't do much with localization so I'm not sure either 🤷‍♂️. I could go either way, my concern is just that we will always have to support that option.